samples/ResourceOwnerPasswordFlow/Startup.cs

```diff
@@ -0,0 +1,255 @@
+﻿
```

Comment:
Don't forget to remove the `ResourceOwnerPasswordFlow` directory if you replace it by `ROPC`